_quarto.yml

```diff
@@ -36,8 +36,10 @@ website:
         menu:
           - text: OpenProblems v1
             href: https://github.com/openproblems-bio/openproblems/blob/main/CONTRIBUTING.md
+            link-external-newwindow: true
```

Comment:
`link-external-newwindow: true` also doesn't work. This cannot be used that's why a specific issue was added to quarto see my issue. this is fixed in quarto 1.3

Reply:
Ah, the problem is our quarto version? Why don't we upgrade?

Reply:
1.3 is in prerelease